asciidoc/workloads/azure/frontend/getting_started.adoc

```diff
@@ -118,6 +118,14 @@ a|Add the playwright testing library and much more to your project!
 - Add visual testing with Applitools eyes
```

Comment:
Do we need to add Cypress on line 81 too?
e.g.
"Create a testing project for the generated Next application. Supported: Playwright & Cypress"

Or is it because Cypress is deprecated?

Reply:
Not needed as no longer supported/ deprecated.